AppCenter/AppCenter/MSUserInformation.h

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSUserInformation : NSObject
```

Comment:
Why are we adding this now when it's just a wrapper for accountId?  We could have waited until we had more user info to store to refactor.  It's probably a good choice to do the refactoring, I just wanted to know the rationale